Dockerfile

```diff
@@ -6,8 +6,9 @@ ARG GO_VERSION=1.13.8
 ARG DEBIAN_FRONTEND=noninteractive
 ARG VPNKIT_VERSION=0.4.0
 ARG DOCKER_BUILDTAGS="apparmor seccomp selinux"
+ARG BUILD_ENV_FROM="golang:${GO_VERSION}-buster"
 
-FROM golang:${GO_VERSION}-buster AS base
+FROM ${BUILD_ENV_FROM} AS base
```

Comment:
I would rename this environment variable to `${GOLANG_IMAGE}` but otherwise looks good to me.

Reply:
Will do